packages/@o3r/testing/schematics/ng-add/templates/workspace/jest.config.ut.js.template

```diff
@@ -14,11 +14,12 @@ globalThis.ngJest = {
  * @returns {import('ts-jest/dist/types').JestConfigWithTsJest}
  */
 module.exports.getJestProjectConfig = (rootDir, isAngularSetup) => {
-  const relativePath = relative(rootDir, '.');
-  const moduleNameMapper = Object.entries(pathsToModuleNameMapper(compilerOptions.paths)).reduce((acc, [moduleName, path]) => {
+  const relativePath = relative(rootDir, __dirname);
+  const moduleNameMapper = Object.entries(pathsToModuleNameMapper(compilerOptions.paths || {})).reduce((acc, [moduleName, path]) => {
     acc[moduleName] = `<rootDir>/${relativePath}/${path}`;
     return acc;
   }, {});
+  moduleNameMapper['^@o3r/testing/core$'] = [require.resolve('@o3r/testing/core/angular')];
```

Comment:
This is also handling the deep imports. No need for extra mappers like `moduleNameMapper['^@o3r/testing/core/(.*)'] = [join(require.resolve('@o3r/testing/core/angular'), '$1')];`

Reply:
Why not doing the change in the tsconfig ?

Reply:
very good question. Because in `tsconfig` we should point to the implementation in `node_modules` but this does not work with `pnp`

Reply:
The regexp does explicitly not match deep import, are you sure it is supported?

Reply:
I don't know how jest works under the hood with `moduleNameMapper` but I did some tests and I am pretty sure deep imports are supported.